src/utils/i18n.js

```diff
@@ -5,12 +5,22 @@
  */
 
 import annotationsLocaleData from 'box-annotations-locale-data'; // eslint-disable-line
-import boxElementsMessages from 'box-elements-messages'; // eslint-disable-line
-import localeData from 'react-intl-locale-data'; // eslint-disable-line
+import boxElementsMessages from 'box-elements-messages';
+import { IntlProvider } from 'react-intl';
 
 declare var __LANGUAGE__: string;
 
 const language = __LANGUAGE__;
-const messages = { ...annotationsLocaleData, ...boxElementsMessages };
+const annotationsMessages = { ...annotationsLocaleData, ...boxElementsMessages };
+const getLocale = (lang: string = __LANGUAGE__) => {
+    return lang.substr(0, lang.indexOf('-'));
+};
 
-export default { language, messages, localeData };
+const createIntlProvider = ({ messages = annotationsMessages, locale = getLocale() }: IntlOptions) => {
+    return new IntlProvider({
+        messages,
+        locale,
+    }).getChildContext().intl;
+};
+
+export default { language, getLocale, createIntlProvider };
```

Comment:
Do you think it would be worthwhile to introduce something like [eslint/sort-keys](https://eslint.org/docs/rules/sort-keys) to help with alphabetization?

Reply:
Yeah, I can enable it in follow up to not conflate this PR with linting changes.